cmd/xl-v1-healing.go

```diff
@@ -639,12 +639,45 @@ func defaultHealResult(storageDisks []StorageAPI, errs []error, bucket, object s
 	return result
 }
 
+// Object is considered dangling/corrupted if any only if
+// total disks - total number of xl.json's are corrupted is lesser than number of data blocks.
+// total disks - total number of xl.json's are missing is lesser than number of data blocks.
```

Comment:
These statements are not accurate and do not reflect the code:

`N - a < k || N - b < k` is not the same as `N - a - b < k`

Reply:
They are simply the gist - the below logic is self-explanatory @donatello what do you suggest that we write?